Makefile

```diff
@@ -38,7 +42,11 @@ sweep:
 default: build
 
 build: fmtcheck
-	go install
+	# trying to copy .goreleaser.yaml
+	go build -a -ldflags '-s -w -extldflags "-static"'
```

Comment:
Given `make build` is primarily used for local development we may want to avoid stripping debug information :slightly_smiling_face: